aiplatform/api/AIPlatform.Samples/PredictTextEmbeddingsSample.cs

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START aiplatform_sdk_embedding]
+
+using Google.Cloud.AIPlatform.V1;
+using wkt = Google.Protobuf.WellKnownTypes;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public class PredictTextEmbeddingsSample
+{
+    // For detailed documentation on text embeddings, see the following:
+    // https://cloud.google.com/vertex-ai/docs/generative-ai/model-reference/text-embeddings
```

Comment:
Where this sample is shown, there' no extra explanation that we need to add this link here?